apps/shark_studio/web/ui/chat.py

```diff
@@ -63,7 +72,7 @@ def chat_fn(
     prefill_time = 0
     is_first = True
     for text, exec_time in language_model.chat(history):
-        history[-1][-1] = text
+        history[-1][-1] = f"{text}{E_SYS}"
```

Comment:
ideally  f"{text}{E_SYS}" would happen at the api layer no?

Reply:
I thought the same but struggled to reproduce good results after moving this to api layer. IIRC it had to do with how we yield from the underlying function call.